src/ucs/sys/sock.c

```diff
@@ -887,25 +887,19 @@ ucs_status_t ucs_sock_port_from_string(const char *port_str, uint16_t *port)
     return UCS_OK;
 }
 
-static int ucs_sockaddr_check_known_af(const struct sockaddr *sa)
-{
-    if (!ucs_sockaddr_is_known_af(sa)) {
-        ucs_error("unknown address family: %d", sa->sa_family);
-        return 0;
-    }
-
-    return 1;
```

Comment:
why removed this function? it seems useful to print the error from it

Reply:
it's no longer related to the PR..

Reply:
ok